geoviews/data/geopandas.py

```diff
@@ -562,14 +566,20 @@ def get_geom_type(geom):
 def to_geopandas(data, xdim, ydim, columns=None, geom='point'):
     """Converts list of dictionary format geometries to spatialpandas line geometries.
```

Comment:
I'm not quite sure this docstring accurately describes what the `to_geopandas` function does.